index.js

```diff
@@ -187,20 +191,19 @@ function readValue(buffer, offset, bigEndian, type) {
 function parseDate(string) {
   if (typeof string !== 'string')
     return null;
-
   var match = string.match(/^(\d{4}):(\d{2}):(\d{2}) (\d{2}):(\d{2}):(\d{2})$/);
   if (!match)
     return null;
 
-  var date = new Date(1970, 0, 2); // to prevent unexpected month change after setUTCMonth()
-  date.setUTCFullYear(match[1]);
-  date.setUTCMonth(match[2] - 1);
-  date.setUTCDate(match[3]);
-  date.setUTCHours(match[4]);
-  date.setUTCMinutes(match[5]);
-  date.setUTCSeconds(match[6]);
-  date.setUTCMilliseconds(0);
-  return date;
+  return new Date(Date.UTC(
```

Comment:
Was this change intentional? It looks like it reverts some of commits https://github.com/devongovett/exif-reader/commit/ea5ed3d307bae40578d820cc5e27d5fcc1ce2c9c and https://github.com/devongovett/exif-reader/commit/be0016f4e4d8dda55c7acaba6276e1f3ccd0c24a

Reply:
Yeah, it was intentional. It fixes the same issue as be0016f4e4d8dda55c7acaba6276e1f3ccd0c24a (that UTC timestamps were interpreted as local time because of `new Date`). If I recall correctly, I made the change to get the test suite to even pass on my computer, which must have been in GMT+2 back on July 16 2016 when I made the change :sweat_smile:

I think converting to UTC in one go is slightly nicer, but either fix does the job. I've undone all the changes to index.js here and made a separate PR now: #15